public/create_user.php

```diff
@@ -35,18 +52,24 @@
 
 <form method="post">
     <label for="first_name">First Name</label>
-    <input type="text" name="first_name" id="first_name">
+    <input type="text" name="first_name" id="first_name" required>
     <label for="last_name">Last Name</label>
-    <input type="text" name="last_name" id="last_name">
-    <label for="password">Password</label>
-    <input type="password" name="password" id="password">
-    <label for="email">Email Address</label>
+    <input type="text" name="last_name" id="last_name" required>
```

Comment:
can probably put type="email" in the input tag for a small form of validation

Reply:
oh good idea